lib/order.rb

```diff
@@ -0,0 +1,31 @@
+class Order
```

Comment:
There are some (useful) methods in here that I don't think are specified in the spec (i.e. being able to remove and clear the basket). I understand why you would implement them but in real world projects I would think you possibly miss delivering against the real spec by going beyond the scope to deliver things you think should be delivered.